docs/finder-content-item.md

```diff
@@ -0,0 +1,61 @@
+# The Finder Content Item Format
+
+A Finder Content Item is a specification of the [Content Item](https://github.com/alphagov/content-store/blob/master/doc/content_item_fields.md). This guide explains what goes in the details hash of the ContentItem and why.
+
+The Finder Content Item is used by Finder Frontend to render the Finder page. Some of the details Most of what it uses goes in the `details` hash.
+
+# The `details` hash
+
+## `beta`
+
+A boolean. Required.
```

Comment:
We could, but I thought I'd pass this on what currently is required and change it when I refactor the App.